Rootless docker images get `permission denied`

Hi,

I am having an issue with the code in your library:

```
/usr/local/lib/python3.11/dist-packages/starlette/formparsers.py:10: in <module>
    import multipart
/usr/local/lib/python3.11/dist-packages/multipart/__init__.py:10: in <module>
    if file_path.is_file():
/usr/lib/python3.11/pathlib.py:1267: in is_file
    return S_ISREG(self.stat().st_mode)
/usr/lib/python3.11/pathlib.py:1013: in stat
    return os.stat(self, follow_symlinks=follow_symlinks)
E   PermissionError: [Errno 13] Permission denied: '/root/multipart.py'
```

Please add handling of this issue!

--- a/multipart/__init__.py
+++ b/multipart/__init__.py
@@ -7,14 +7,17 @@ from pathlib import Path
 
 for p in sys.path:
     file_path = Path(p, "multipart.py")
-    if file_path.is_file():
-        spec = importlib.util.spec_from_file_location("multipart", file_path)
-        assert spec is not None, f"{file_path} found but not loadable!"
-        module = importlib.util.module_from_spec(spec)
-        sys.modules["multipart"] = module
-        assert spec.loader is not None, f"{file_path} must be loadable!"
-        spec.loader.exec_module(module)
-        break
+    try:
+        if file_path.is_file():
+            spec = importlib.util.spec_from_file_location("multipart", file_path)
+            assert spec is not None, f"{file_path} found but not loadable!"
+            module = importlib.util.module_from_spec(spec)
+            sys.modules["multipart"] = module
+            assert spec.loader is not None, f"{file_path} must be loadable!"
+            spec.loader.exec_module(module)
+            break
+    except PermissionError:
+        pass
 else:
     warnings.warn("Please use `import python_multipart` instead.", PendingDeprecationWarning, stacklevel=2)
     from python_multipart import *

